src/main/java/no/nav/familie/integrasjoner/config/RestTemplateConfig.kt

```diff
@@ -67,6 +69,21 @@ class RestTemplateConfig(
         }
     }
 
+    @Bean("jwt-sts")
+    fun restTemplateJwtBearerFallbackSts(bearerTokenClientInterceptor: BearerTokenWithSTSFallbackClientInterceptor,
```

Comment:
Forventer dokarkiv både sts og SB token?

Reply:
Ihvertfall STS, for det er det som er brukt før. SB-token er hva vi tester nå.

Reply:
Er kanskje navnet litt misledende isåfall? Burde kanskje vært `azure-sts-fallback`?